### PR TITLE
pep8 fixes

### PR DESCRIPTION
**Pull request by kaedroho**
This fixes some pep8 issues that have popped after moving to the new CI server (which caused flake8 to update).

### CODE REVIEW

File: wagtail/wagtailembeds/oembed_providers.py [hunk 1]
```diff
@@ -301,7 +304,6 @@
 
 
 # Compile endpoints into regular expression objects
-import re
```
Comment (gasman) [inline wagtail/wagtailembeds/oembed_providers.py]: Would make sense to remove these blank lines too.

File: wagtail/wagtailsearch/backends/base.py [hunk 2]
```diff
@@ -64,8 +64,8 @@ def _process_filter(self, field_attname, lookup, value):
 
         if result is None:
             raise FilterError(
-                'Could not apply filter on search results: "' + field_attname + '__'
-                + lookup + ' = ' + text_type(value) + '". Lookup "' + lookup + '"" not recognosed.'
+                'Could not apply filter on search results: "' + field_attname + '__' +
+                lookup + ' = ' + text_type(value) + '". Lookup "' + lookup + '"" not recognosed.'
```
Comment (gasman) [inline wagtail/wagtailsearch/backends/base.py]: 'recognosed'? :-) (Not related to the PEP8 fixes I know, but worth fixing too I dare say...)

**Comment by kaedroho** [comment]
Cheers for the quick review!

**Comment by gasman** [comment]
:arrow_heading_down: in ca6c43df4836b436d3ed69a844fcb4ba13fc6489

**Pull request closed without merging**